src/components/elements/wysiwyg.tsx

```diff
@@ -80,7 +81,11 @@ const options: HTMLReactParserOptions = {
           return cleanMediaMarkup(domNode)
 
         case "p":
-          nodeProps.className = twMerge(nodeProps.className, "max-w-[100ch] leading-[1.7] text-21")
+          nodeProps.className = twMerge(
+            nodeProps.className,
+            "max-w-[100ch]",
+            clsx({"type-0": !nodeProps.className.includes("type-")})
```

Comment:
`nodeProps` might not contains `className`. It might even be undefined.

```suggestion
            clsx({"type-0": !nodeProps?.className/.includes("type-")})
```

Reply:
Makes sense to me! 😎